<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bar Chart </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout">
   <item row="10" column="0">
    <widget class="QCheckBox" name="fixPlaneSizeCB">
     <property name="text">
      <string>Data / plane space relation fixed</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>171</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="widthLB">
       <property name="text">
        <string>Configure Width</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="widthCB">
         <property name="text">
          <string>Bars Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="widthSB"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="threeDLB">
       <property name="text">
        <string>Paint ThreeD Bars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="paintThreeDBarsCB">
       <property name="text">
        <string>ThreeD</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="threeDDepthCB">
         <property name="text">
          <string>Bars Depth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="depthSB">
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="penLabel">
       <property name="text">
        <string>Pen Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="markColumnCB">
         <property name="text">
          <string>Mark Column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="markColumnSB">
         <property name="maximum">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="barTypeLB">
       <property name="text">
        <string>Bar Chart Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="barTypeCB">
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stacked</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Percent</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="barOrientationLB">
       <property name="text">
        <string>Bar Orientation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="barOrientationCB">
       <item>
        <property name="text">
         <string>Vertical</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="paintValuesCB">
     <property name="text">
      <string>Display Data Values</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="12">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="chartFrame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="BarChart.qrc"/>
 </resources>
 <connections/>
</ui>
